lib/main.coffee

```diff
@@ -28,11 +33,37 @@ module.exports =
       lint: (textEditor) =>
         filePath = textEditor.getPath()
         fileText = textEditor.getText()
+        fileDir = _path2.default.dirname(filePath)
+
+        configFile = helpers.findCached fileDir, [
```

Comment:
I was going off [their documentation](http://www.html-tidy.org/documentation/#using-a-configuration-file) which says the absolute path of a configuration file _can_ (but isn't required to) be specified using htat environment variable.

Since this PR is all about adding the configuration file support, I figured now would be a good time to implement that as well.